detox/src/ios/expectTwo.test.js

```diff
@@ -754,6 +757,42 @@ describe('expectTwo', () => {
     expect(testCall).toDeepEqual(jsonOutput);
   });
 
+  describe('system', () => {
+    it(`should parse system.element(by.system.label('tapMe')).atIndex(1).tap()`, async () => {
+      const testCall = await e.system.element(e.by.system.label('tapMe')).atIndex(1).tap();
+      const jsonOutput = {
+        invocation: {
+          type: 'systemAction',
+          systemAction: 'tap',
+          systemPredicate: {
```

Comment:
I wonder what caused you to rename literally every property inside the object?

On the first glance, an object like 

```js
{
  type: 'systemAction',
  action: 'tap',
  predicate: {
   // ...
  },
  atIndex: 1
}
```

...could have perfectly co-exist with other invocations. Wouldn't `type: 'systemAction'` be enough already for distinguishing those?

Reply:
You make a good point, and I had considered that too. However, I chose to follow the naming "conventions" established in the WebView APIs (`webAction`, `webPredicate`, `webExpectation`..) to maintain a consistent design approach